### ISSUE TITLE
"Private members is used" does not understand VB's Handles pattern

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn-analyzers#1872]
#### Diagnostic ID

Example: [CA1716](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1716-identifiers-should-not-match-keywords)

#### Repro steps

``` VB
        Private Sub Cancel_Click(sender As Object, e As EventArgs) Handles Cancel.Click
            _loadAnonymous = True
            DialogResult = DialogResult.OK
        End Sub
```
#### Expected behavior
No message

#### Actual behavior

```
Severity	Code	Description	Project	File	Line	Suppression State
Message	IDE0051	Private member 'ServicesAuthenticationForm.Cancel_Click' is unused.	Microsoft.VisualStudio.Editors	E:\project-system2\src\Microsoft.VisualStudio.Editors\PropPages\ServicesAuthenticationForm.vb	72	Active
```
